int8    some_number
Test[2] test_static
Test[]  test_dynamic
int8    another_number
string  aString
string[2] sStrings
string[] dStrings
